src/EnergyPlus/PlantComponent.hh

```diff
@@ -59,7 +59,7 @@ class PlantComponent
 {
 
 public:
-    virtual void simulate(const PlantLocation &calledFromLocation, bool const FirstHVACIteration, Real64 &CurLoad, bool const RunFlag) = 0;
+    virtual void simulate(const PlantLocation &calledFromLocation, bool FirstHVACIteration, Real64 &CurLoad, bool RunFlag) = 0;
```

Comment:
This is a fine change to clean out the warnings in the IDE about header constness.